src/generate-from-template.js

```diff
@@ -32,8 +33,16 @@ const generateFromTemplate = async ({ templateName }) => {
   // eslint-disable-next-line import/no-extraneous-dependencies,global-require,import/no-dynamic-require
   const templatePackage = require(templatePackageName);
 
+  let templateBanner;
+  if (typeof templatePackage.getTemplateBanner === 'function') {
+    const templateBannerResponse = templatePackage.getTemplateBanner(kleur);
+    if (typeof templateBannerResponse === 'string') {
```

Comment:
If its not a string, should we throw an error so the user knows why its not working ?

Reply:
We would want the template developer to know, but I'm not sure we would want to show an error to the template user?
Part of the banner being missing is not a critical failure?

Reply:
Good point, this isn't critical. I think it can be left as is